skymap_scanner/server/utils.py

```diff
@@ -159,20 +164,31 @@ def __hash__(self) -> int:  # type: ignore[override]
 
     @cachetools.func.lru_cache()
     def n_recos_by_nside_lowerbound(self, n_posvar: int) -> Dict[int, int]:
-        """Get estimated # of recos per nside.
+        """Get # of recos per nside (w/ predictive scanning its a LOWER bound).
 
-        These are ESTIMATES (w/ predictive scanning it's a LOWER bound).
+        The actual pixel generation is done iteratively, and does not
+        rely on this function. Use this function for reporting & logging
+        only.
         """
         int_int_list = self._get_int_int_list()
 
```

Comment:
Would it not be simpler to retain the previous code and just create a new `int_int_list` with `12` in the first tuple?  I'm not sure the additional functions improve the readability, and both are just trying to catch the same case.

Reply:
That's a great point. I didn't think about that